new lint error MissingSuperCall

## Bug description 
With the update of androidx.core in #10680 there are found two new lint errors:
```
<!-- Enter a summarized description of what the bug/problem is, that you found -->
  src/cgeo/geocaching/InstallWizardActivity.java:417: Error: Overriding method should call super.onRequestPermissionsResult [MissingSuperCall]
      public void onRequestPermissionsResult(final int requestCode, @NonNull final String[] permissions, @NonNull final int[] grantResults) {
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~
  src/cgeo/geocaching/MainActivity.java:370: Error: Overriding method should call super.onRequestPermissionsResult [MissingSuperCall]
      public void onRequestPermissionsResult(final int requestCode, @NonNull final String[] permissions, @NonNull final int[] grantResults) {
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~
```  
This has to be solved before updating of these library.

--- a/main/src/cgeo/geocaching/InstallWizardActivity.java
+++ b/main/src/cgeo/geocaching/InstallWizardActivity.java
@@ -415,6 +415,7 @@ public class InstallWizardActivity extends AppCompatActivity {
 
     @Override
     public void onRequestPermissionsResult(final int requestCode, @NonNull final String[] permissions, @NonNull final int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
         gotoNext();
     }
 

--- a/main/src/cgeo/geocaching/MainActivity.java
+++ b/main/src/cgeo/geocaching/MainActivity.java
@@ -368,6 +368,7 @@ public class MainActivity extends AbstractActionBarActivity {
 
     @Override
     public void onRequestPermissionsResult(final int requestCode, @NonNull final String[] permissions, @NonNull final int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
         if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
             PermissionHandler.executeCallbacksFor(permissions);
         } else {

